manifests/0000_31_cluster-baremetal-operator_03_baremetalhost.crd.yaml

```diff
@@ -1,13 +1,12 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: baremetalhosts.metal3.io
   annotations:
-    include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
```

Comment:
@honza did you manually merge these in the previous update? 

`get-bmh-crd.sh`
 doesn't add them, so I'm not sure where these come from, or why we need them tbh.

Reply:
Ack, thanks, I guess we could merge these in the hack script with `yq` but it'd be good to clarify where we look to keep the openshift-specific additions updated first so I'll do it manually for now :)